docs/source/contributing.rst

```diff
@@ -314,13 +314,16 @@ An important example of that is ``download_utils.RemoteFileMetadata``. Please us
 Make sure to include, in the docstring of the dataloader, information about the following list of relevant aspects about the dataset you are integrating:
 
 * The dataset name
-
 * A general purpose description, the task it is used for
 * Details about the coverage: how many clips, how many hours of audio, how many classes, the annotations available, etc.
 * The license of the dataset (even if you have included the ``LICENSE_INFO`` variable already)
 * The authors of the dataset, the organization in which it was created, and the year of creation (even if you have included the ``BIBTEX`` variable already)
 * Please reference also any relevant link or website that users can check for more information
 
+.. note::  
+    In addition to the module docstring, you should also write a docstrings for your Clip class, and for the functions you write. See :ref:`the documentation tutorial <documentation_tutorial>` for practical information on best documentation practices.
```

Comment:
you should also write a docstrings for your Clip class, and for the functions you write. --> you should write docstrings for every function you write.?